src/test/developTest/java/mServer/crawler/sender/newsearch/ZdfDownloadDTODeserializerTest.java

```diff
@@ -12,41 +12,51 @@
-        this.subtitleUrl = subtitleUrl;
-        this.smallQualityUrl = smallQualityUrl;
-        this.normalQualityUrl= normalQualityUrl;
-        this.hdQualityUrl = hdQualityUrl;
-    }
-    
-    @Test
-    public void testDeserialize() {
-        
-        JsonObject jsonObject = JsonFileReader.readJson(jsonFile);
-        
-        ZDFDownloadDTODeserializer target = new ZDFDownloadDTODeserializer();
-        DownloadDTO actual = target.deserialize(jsonObject, DownloadDTO.class, null);
-        
-        assertThat(actual, notNullValue());
-        assertThat(actual.getSubTitleUrl(), equalTo(subtitleUrl));
-        assertThat(actual.getUrl(Qualities.SMALL), equalTo(smallQualityUrl));
-        assertThat(actual.getUrl(Qualities.NORMAL), equalTo(normalQualityUrl));
-        assertThat(actual.getUrl(Qualities.HD), equalTo(hdQualityUrl));
-    }
+
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][]{
+      {"/zdf/zdf_download_sample_with_subtitle_nohd.json", "https://utstreaming.zdf.de/mtt/zdf/17/02/170212_sendung2_sof/3/F1011017_hoh_deu_Lindstroem_Liebesreigen_in_Samlund_120217.xml", "https://rodlzdf-a.akamaihd.net/none/zdf/17/02/170212_sendung2_sof/3/170212_sendung2_sof_476k_p9v13.mp4", "https://rodlzdf-a.akamaihd.net/none/zdf/17/02/170212_sendung2_sof/3/170212_sendung2_sof_1496k_p13v13.mp4", "", "", "", ""},
+      {"/zdf/zdf_download_sample_without_subtitle_hd.json", "", "https://nrodlzdf-a.akamaihd.net/de/zdf/17/02/170212_kompakt_spo/1/170212_kompakt_spo_229k_p7v13.mp4", "https://rodlzdf-a.akamaihd.net/de/zdf/17/02/170212_kompakt_spo/1/170212_kompakt_spo_1496k_p13v13.mp4", "https://rodlzdf-a.akamaihd.net/de/zdf/17/02/170212_kompakt_spo/1/170212_kompakt_spo_476k_p9v13.mp4", "", "", ""},
+      {"/zdf/zdf_download_sample_with_english.json", "https://utstreaming.zdf.de/mtt/zdf/18/04/180416_2215_sendung_hsn/7/Hard_Sun_Teil1_OmU.xml", "https://rodlzdf-a.akamaihd.net/de/zdf/18/04/180416_2215_sendung_hsn/7/180416_2215_sendung_hsn_a1a2_476k_p9v13.mp4", "https://rodlzdf-a.akamaihd.net/de/zdf/18/04/180416_2215_sendung_hsn/7/180416_2215_sendung_hsn_a1a2_1496k_p13v13.mp4", "", "https://rodlzdf-a.akamaihd.net/de/zdf/18/04/180416_2215_sendung_hsn/7/180416_2215_sendung_hsn_a3a4_476k_p9v13.mp4", "https://rodlzdf-a.akamaihd.net/de/zdf/18/04/180416_2215_sendung_hsn/7/180416_2215_sendung_hsn_a3a4_1496k_p13v13.mp4", ""}
+    });
+  }
+
+  private final String jsonFile;
+  private final String subtitleUrl;
+  private final String smallQualityUrl;
+  private final String normalQualityUrl;
+  private final String hdQualityUrl;
+  private final String smallQualityUrlEng;
+  private final String normalQualityUrlEng;
+  private final String hdQualityUrlEng;
+
+  public ZdfDownloadDTODeserializerTest(String jsonFile, String subtitleUrl, String smallQualityUrl, String normalQualityUrl, String hdQualityUrl, String smallQualityUrlEng, String normalQualityUrlEng, String hdQualityUrlEng) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)